RussianFilter/sections/css_extended.txt

```diff
@@ -306,6 +306,8 @@ kulturologia.ru#?##main div.js-relap__item:has(> div.js-relap__item-inner > a[hr
 !
 soft.mydiv.net#?#div[id^="downfile_div"] > div[style]:has(> div > div[style]:contains(Попробуйте также))
 1plus1.video#?#.video-index > div[class]:has(> div[class$="_ad"])
+udf.by#?#.article_list > div.helper_line > div.article_list_in > h2:contains(Новости других СМИ)
+udf.by#?##content > div#dle-content > div.article_list:has(> div[id^="adfox"])
```

Comment:
First ext-css rule to`udf.by##.helper_line > div.article_list_in:last-child`. The last you can re-write to `udf.by#?###dle-content > div.article_list:has(> div[id^="adfox"])`, also add domain to `Russian -> specific` `##div[id^="adfox_"]`